xgboost_caip_e2e.ipynb

```diff
@@ -19,52 +19,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 0,
+   "execution_count": null,
    "metadata": {
     "colab": {},
     "colab_type": "code",
     "id": "hSFHu19Rtvbt"
    },
    "outputs": [],
    "source": [
-    "# If you're running on Cloud AI Platform notebooks, you'll need to install xgboost and restart the runtime\n",
-    "# !pip3 install xgboost"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 0,
-   "metadata": {
-    "colab": {},
-    "colab_type": "code",
-    "id": "G-B8Q-4W9X6q"
-   },
-   "outputs": [],
-   "source": [
-    "# You can skip this cell if you're running this from Cloud AI Platform Notebooks\n",
+    "# If you're running on Colab, you'll need to install the What-if Tool package and authenticate\n",
+    "# If you're on Cloud AI Platform Notebooks, you'll need to install XGBoost on the TF instance\n",
+    "import os\n",
     "\n",
-    "# This is the Python package you'll use to access the What-if Tool (WIT)\n",
-    "# You can ignore the prompt-toolkit error from this pip install\n",
-    "!pip install witwidget --quiet"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 0,
-   "metadata": {
-    "colab": {},
-    "colab_type": "code",
-    "id": "PzmV_odNPBTb"
-   },
-   "outputs": [],
-   "source": [
-    "# You can skip this cell if you're running this from Cloud AI Platform Notebooks\n",
+    "if 'COLAB_GPU' in os.environ:\n",
```

Comment:
is this true even when running in colab without gpu availability? i've seen another way to test for colab by i think trying to import google.colab or something, and catching any error in which case it is not colab.

Reply:
```
try:
  import google.colab
  IN_COLAB = True
except:
  IN_COLAB = False
```

Reply:
That worked! Updated.